flair/data.py

```diff
@@ -337,19 +337,139 @@ def __repr__(self) -> str:
         )
 
 
+def space_tokenizer(text: str) -> List[Token]:
+    """
+    Tokenizer based on space character only.
+    """
+    tokens: List[Token] = list()
```

Comment:
In the code, the convention `[]` is used, maybe we should keep it that way for consistency ?